micropub.php

```diff
@@ -136,11 +139,8 @@ public static function parse_query( $wp ) {
 		}
 		static::$input = apply_filters( 'before_micropub', static::$input );
 
-		// For debug purposes be able to bypass Micropub auth with WordPress auth
+		// Be able to bypass Micropub auth with other auth
 		if ( MICROPUB_LOCAL_AUTH ) {
-			if ( ! is_user_logged_in() ) {
-				auth_redirect();
```

Comment:
is this unnecessary? why?

Reply:
The auth redirect never worked properly.

Reply:
oh! um, ok. do we need it? i guess micropub requests generally aren't interactive, so this isn't useful anyway...? do we currently return HTTP 401 when `MICROPUB_LOCAL_AUTH` is set but the user isn't logged in?

Reply:
That I will check